pom.xml

```diff
@@ -4,7 +4,7 @@
 
   <groupId>org.nuxeo.ai</groupId>
   <artifactId>nuxeo-insight-client</artifactId>
-  <version>1.1.1-SNAPSHOT</version>
+  <version>1.2.0-SNAPSHOT</version>
```

Comment:
just a note for the future, this should be a PR on its own.

Reply:
ok